docs/swagger.yaml

```
@@ -16,19 +17,41 @@ definitions:
+        enum:
+        - hak
+        - mupadmin
+        - osoba
+        - firma
+        - policija
+        - superadmin
         type: string
       uuid:
         type: string
+    required:
+    - birthDate
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)